Makefile

```diff
@@ -54,7 +54,7 @@ define run-in-test-server
 		-v ${GOCACHE}:/tmp/go-build-cache${VOLUME_MOUNT} \
 		-v ${HOME}/.kube:/home/user/.kube${VOLUME_MOUNT} \
 		-v /tmp:/tmp${VOLUME_MOUNT} \
-		-v /app/config:/app/config${VOLUME_MOUNT} \
+		-v /tmp/argo-e2e/app/config:/app/config${VOLUME_MOUNT} \
```

Comment:
Hm, I think this is not the correct fix. `/tmp/argo-e2e` only exists during E2E tests, and lives within the container. This mount is for running ArgoCD locally, and it should persist the application configuration (TLS certs, SSH known hosts and also GPG keys in the future) somewhere. But I agree, `/app/config` might not be the correct place on the host.

Reply:
After thinking about it for a while, I think we can remove the mount all together - if `/app/config` needs updating within the container, it should be done using `docker cp` - the data does not need to be persistent.